rllib/core/rl_module/tf/tests/test_tf_rl_module.py

```diff
@@ -1,14 +1,16 @@
-import gymnasium as gym
-import tensorflow as tf
-import tensorflow_probability as tfp
 import unittest
 from typing import Mapping
 
+import gymnasium as gym
+import tensorflow as tf
```

Comment:
Nit: Let's also use `try_import_tf()` here. I know it's just a test case, but also `tensorflow` is NOT a dependency for RLlib, so it might now be installed on a user's machine when they run this test.

Reply:
Absolutely, I had not yet scanned the repo for such tensorflow imports!
I've done that now and added these to this PR.